silta-cicd/circleci-php8.2-node18-composer2/README.md

```diff
@@ -0,0 +1,14 @@
+# silta-circleci
+A docker image used circleCI, based on `cimg/php:8.1.7-node` with the following additions:
+
+- Composer configured correctly
+- Drush-launcher and coder pre-installed
+- Vim, useful for debugging
+- The google cloud cli, kubernetes and helm
+
+## Versions
+- PHP: 8.1.7
```

Comment:
Should this be 8.2.x?

Reply:
fixed it, then figured out google sdk version is supposed to be deprecated